compiler/rustc_codegen_ssa/src/mir/rvalue.rs

```diff
@@ -216,6 +216,10 @@ impl<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>> FunctionCx<'a, 'tcx, Bx> {
                         // This is a no-op at the LLVM level.
                         operand.val
                     }
+                    mir::CastKind::Pointer(PointerCast::DeprecatedSafeFnPointer) => {
+                        // This is a no-op at the LLVM level.
+                        operand.val
+                    }
```

Comment:
could merge this with the branch above instead of copying it

Reply:
done